packages/fluentui/react-northstar/src/themes/teams/components/Avatar/avatarStatusIconStyles.ts

```diff
@@ -3,7 +3,7 @@ import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
 import { AvatarStatusStylesProps } from '../../../../components/Avatar/AvatarStatus';
 import { AvatarVariables } from './avatarVariables';
 
-export const avatarStatusStyles: ComponentSlotStylesPrepared<AvatarStatusStylesProps, AvatarVariables> = {
+export const avatarStatusIconStyles: ComponentSlotStylesPrepared<AvatarStatusStylesProps, AvatarVariables> = {
```

Comment:
This is just a forgotten thing from the previous PR right ?

Reply:
Yep